modules/installation-osp-creating-network-resources.adoc

```diff
@@ -54,3 +60,17 @@ $ ansible-playbook -i inventory.yaml 02_network.yaml
 ----
 $ openstack subnet set --dns-nameserver <server_1> --dns-nameserver <server_2> "$INFRA_ID-nodes"
 ----
+. Optional: If you did not provide a value for `os_external_network` in the `inventory.yaml` file, on a command line, run the following script to create a router and attach a control-plane floating IP address to it:
```

Comment:
I don't think that the users are interested in these instructions. These lines literally do exactly what the scripts would do if you pass to them "os_external_network".

Instead, I'd say something like "If you did not provide a value for `os_external_network`, you need yourself to provide the VMs access to Glance and external connectivity"